### DISCUSSION TITLE
Thumbnails Error

### DISCUSSION THREAD

**Discussion by Lanceman56** [stumpapp/stump#459] [Question] [answered]
I am getting this error when I add new books to my library: `Failed to generate thumbnail: "LoadLibraryError(\n DlSym {\n desc: \"/lib/libpdfium.so: undefined symbol: FPDFText_GetTextObject\",\n },\n)"` 
I have pulled the latest container and this still happens.

**Comment by aaronleopold** [comment, marked as answer]
This was discussed on [Discord](https://discord.com/channels/972593831172272148/972595055061794856/1284566941477896305). The fix is in the `nightly` image, but will otherwise be released in `0.0.7`

A workaround would be manually linking a local and up-to-date PDFium to your container via a mount and point it to it, but I haven't personally tried it.

**Reply by Lanceman56** [reply]
Thanks!
